CONTRIBUTING.md

```diff
@@ -52,3 +52,14 @@ guard
 ## Daru internals
 
 To get an overview of certain internals of daru and their implementation, go over [this blog post](http://v0dro.github.io/blog/2015/08/16/elaboration-on-certain-internals-of-daru/).
+
+
+## Troubleshooting
```

Comment:
I think another section is not needed. You can make a `Note`  around line- 24:  https://github.com/SciRuby/daru/pull/504/files#diff-6a3371457528722a734f3c51d9238c13R24 , stating the packable issue for running `bundle exec rspec` for Ruby version >= 2.5 . Then you can mention `To reproduce this issue or explore this error further, head over to [issue #503](https://github.com/SciRuby/daru/issues/503).`